src/components/Modals/ReceivedRequestModal.tsx

```diff
@@ -44,19 +64,78 @@ const ReceivedRequestModal = (props: ReceivedModalProps): JSX.Element => {
     onClose();
     addToast(
       props.otherUser.preferredName +
-        "'s request to carpool with you has been rejected.",
+        "'s request to carpool with you has been deleted.",
       { appearance: "success" }
     );
   };
 
+  const validateRequestAcceptance = () => {
+    if (props.user.role == "DRIVER") {
+      if (props.user.seatAvail === 0) {
+        addToast(
+          "You do not have any space in your car to accept " +
+            props.otherUser.preferredName
+        );
+        return false;
+      }
+      if (props.otherUser.carpoolId) {
+        addToast(
+          props.otherUser.preferredName +
+            " is already in an existing carpool group. Ask them to leave that group before attempting to join yours"
```

Comment:
Also very minor but should we add periods to the end of the messages used in this component?